package.json

```diff
@@ -76,7 +76,7 @@
     "lint-staged": "^5.0.0",
     "mocha": "^3.0.0",
     "mochainon": "^2.0.0",
-    "prettier": "^1.8.2",
+    "prettier": "^1.11.1",
```

Comment:
Just wondering: is there any reason we are not using package-lock in this project as well?

Reply:
We shouldn't use lock files on most libraries, since the lock files do not reach our users & we will not be testing against the actual node modules that our consumers get with an `npm i balena-sdk`. For example, let's say that `memoizee` 0.4.10 regresses and our lock file is on 0.4.9, b/c of the package.json being on "^0.4.9", we will never notice that the sdk is broken for our consumers.
If there were "devDependencies only" lock files, we could use them though to make our ci more predictable.

Reply:
:+1: true, haven't realised this is a library. Maybe the strategy is to keep a lockfile for our CI (since we would need to rerun our CI for every deep dependency update anyway), don't publish it and keep transitive dependencies in check with some tool like greenkeeper